specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -866,6 +866,14 @@
         "requireInfrastructureEncryption": {
           "$ref": "#/definitions/WorkspaceCustomBooleanParameter",
           "description": "A boolean indicating whether or not the DBFS root file system will be enabled with secondary layer of encryption with platform managed keys for data at rest."
+        },
+        "storageAccountName": {
+          "$ref": "#/definitions/WorkspaceCustomStringParameter",
+          "description": "Name of the default DBFS storage account name."
+        },
+        "storageAccountSkuName": {
+          "$ref": "#/definitions/WorkspaceCustomStringParameter",
+          "description": "Storage account SKU name"
```

Comment:
Along with above parameter. Can you also add following NPIP Properties under    "parameters": {} section.
 "natGatewayName": {
                "type": "String",
                "value": "nat-gateway"
            },
   "publicIpName": {
         "defaultValue": "nat-gw-public-ip",
         "metadata": {
            "description": "Name of the Public IP associated with the NAT gateway"
         },
         "type": "String"
      },

Reply:
Are these being discussed with PM and got approval for exposing them?

Reply:
yes. it is one of the internal customer ask.

Reply:
can you share me email thread and relevant S360 items as well to update breaking change API request.

Reply:
Added the parameters as per the email thread for No Public IP.